accumulator/starter/main.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"math/rand"
+
+	accumulator "github.com/temporalio/samples-go/accumulator"
+	"go.temporal.io/sdk/client"
+)
+
+var WorkflowIDPrefix = "accumulate"
+
+var TaskQueue = "accumulate_greetings"
+
+func main() {
+	// The client is a heavyweight object that should be created once per process.
+	c, err := client.Dial(client.Options{
+		HostPort: client.DefaultHostPort,
+	})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	// setup which tests to run
```

Comment:
You are inconsistent with whether you capitalize/punctuate sentences. Also, this is a bit confusing to follow.